torch/_dynamo/variables/builtin.py

```diff
@@ -581,7 +581,28 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
-    def _call_min_max(self, tx, a, b):
+    def _call_min_max(self, tx, *args):
+        if len(args) == 1 and args[0].has_unpack_var_sequence(tx):
+            return self._call_min_max_iterable(tx, args[0])
+        elif len(args) == 2:
+            return self._call_min_max_binary(tx, args[0], args[1])
+        else:
+            unimplemented(
+                f"unsupported min / max over {len(args)} args: {[str(a) for a in args]}"
+            )
```

Comment:
This is a bit redundant with the else / fallthrough case below, doesn't matter too much tho

Reply:
True, it's not even that much nicer of a message. I'll remove it